src/zenml/zen_stores/migrations/versions/1d8f30c54477_migrate_to_new_.py

```diff
@@ -0,0 +1,120 @@
+"""migrate to new ModelVersionDataLazyLoader structures [1d8f30c54477].
+
+Revision ID: 1d8f30c54477
+Revises: 0.67.0
+Create Date: 2024-10-08 15:55:34.727543
+
+"""
+
+import json
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "1d8f30c54477"
+down_revision = "0.67.0"
+branch_labels = None
+depends_on = None
+
+
```

Comment:
I think theres one legacy column still missing in this migration: `PipelineRunSchema.pipeline_configuration`

Reply:
I don't think so - this field has no Step config in it, so there is nothing to update, IMO.

Reply:
Ah you're right, I was confusing it with the `Model` object that is defined in there, but that's not relevant. Sorry!